dashboard-notifications/src/test/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.dashboard.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.dashboard.repositories.NotificationRepository;
+
+import java.util.UUID;
+
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class DashboardNotificationServiceTest {
+
+    private DashboardNotificationService dashboardNotificationService;
+
+    @Mock
+    private NotificationRepository notificationRepository;
+
+    private final UUID id = UUID.randomUUID();
+
+    @BeforeEach
+    void setUp() {
```

Comment:
Can we use @InjectMocks rather than Initializing the service before every test ? Example can be found in TaskListServiceTest. I am also happy to leave it as is

Reply:
Done